src/sql/workbench/services/objectExplorer/browser/treeCreationUtils.ts

```diff
@@ -46,7 +46,7 @@ export class TreeCreationUtils {
 		const actionProvider = instantiationService.createInstance(ServerTreeActionProvider);
 		const renderer = instantiationService.createInstance(ServerTreeRenderer, false);
 		const controller = instantiationService.createInstance(ServerTreeController, actionProvider);
-		const dnd = instantiationService.createInstance(ServerTreeDragAndDrop);
+		const dnd = instantiationService.createInstance(TreeDragAndDrop);
```

Comment:
You can let the name be what it was because it was more specific. This drag and drop controller is specific to servers.